ark/utils/segmentation_utils.py

```diff
@@ -108,6 +108,9 @@ def transform_expression_matrix(cell_table, transform, transform_kwargs=None):
     valid_transforms = ['size_norm', 'arcsinh']
     misc_utils.verify_in_list(transform=transform, valid_transforms=valid_transforms)
 
+    if transform not in valid_transforms:
```

Comment:
You can now completely delete this line and the line below it, since we already accomplish the same thing with `misc_utils.verify_in_list`